### ISSUE TITLE
[Feature] Windows store RT distro 

### ISSUE THREAD

**Issue by dvorka** [dvorka/mindforger#1531]
**Enhancement / feature description:**

Prepare Windows store RT distribution for accounts without administrator permissions. 

Would an archive with binary, libs and startup script be an option? 

...

**Comment by listrdave** [comment]
Yes, it will be great if script will be powershell or some launching exe file without any installation step on Windows.
